packages/nextra/src/types.ts

```diff
@@ -60,3 +80,7 @@ export type NextraConfig = {
 export type Nextra = (
   ...args: [NextraConfig] | [theme: Theme, themeConfig: string]
 ) => (nextConfig: NextConfig) => NextConfig
+
+const nextra: Nextra = (...args) => (nextConfig) => ({})
+
+export default nextra
```

Comment:
why it's needed?

Reply:
The key problem here is that we want expose all types which are located in `types.ts` but you can't do that with commonjs.

So i compile `types.ts` to a d.ts file and use packjson's `types` field to let typescript resolve the correct type here. 

```ts
// I have to provider a fake function here otherwise typescript will think i only import a type here
import nextra from 'nextra'
```